### DISCUSSION TITLE
Upgrading from webpacker v5 to shakapacker v6 renders a lot more pack tags and IE11 doesn't work

### DISCUSSION THREAD

**Discussion by chriszimmerman-parachute** [shakacode/shakapacker#248] [Q&A] [answered]
Hello,

I am following [the migration guide](https://github.com/shakacode/shakapacker/blob/master/docs/v6_upgrade.md) for upgrading Webpacker v5 to Shakapacker v6.5.5 and I have several questions:

1. I noticed that I am seeing a lot more pack tags rendered on the webpage now that I have upgraded to shakapacker v6.5.5. Pre-upgrade, I had just one pack (let's call it `myApp`), and now I have `myApp` in addition to a bunch of pack tags who names appear to be a bunch of concatenated `node_module` names. 

Is this to be expected? If so, can someone give some insight on how these packs are generated? It looks like I had a large `myApp` pack pre-upgrade. Now I have a much smaller `myApp` pack with a bunch of small `node_modules` packs.

2. My app works with IE11 pre-upgrade, but now I get a bunch of syntax errors in IE 11 in those new `node_modules` packs. I have not changed anything significant in any of my configs (my babel.config.js comes to mind). I think they need to be transpiled to ES5 since the syntax errors involve things like arrow functions and the presence of the `class` keyword.
 
I am using the `browserslist` config to point to IE11 and I am importing `core-js/stable` and `regenerator-runtime/runtime` at the top of my pack entry points - I would think that would take care of the transpilation.

Is there something I'm missing? Any help would be appreciated!

**Comment by justin808** [comment, marked as answer]
> Is this to be expected? If so, can someone give some insight on how these packs are generated? It looks like I had a large myApp pack pre-upgrade. Now I have a much smaller myApp pack with a bunch of small node_modules packs.

Yes, expected due to the default being code-splitting.
